gcs/CHANGES.md

```diff
@@ -35,6 +35,8 @@
     fs.gs.cooperative.locking.max.concurrent.operations
     ```
 
+1.  Add support for refresh token grant flow.
```

Comment:
Add a little more details how to use it (just short mention, not as specific as config doc).